httptrace/server.go

```diff
@@ -100,7 +100,7 @@ func Middleware(c appdash.Collector, conf *MiddlewareConfig) func(rw http.Respon
 		if e.Route != "" {
 			rec.Name(e.Route)
 		} else {
-			rec.Name(e.Request.Host)
+			rec.Name(r.URL.Host + r.URL.Path)
```

Comment:
Right, `e.Request` is actually not an HTTP request like you'd expect just from a glance. `r` is the real HTTP request, the other (`e`) is just a custom "request info" structure that does not contain the parsed URL (only host), so the switch is needed.